### PR TITLE
[Coinpaprika] new feature: favorite coins

### PR DESCRIPTION
**Pull request by gumbleliks**
## Description

**New feature - Favorite Coins**
- Additional Section -> Favorite Coins
- Add & Remove Favorite Coins
- Action Show & Hide Favorite Coins in Coin List

## Screencast
showcase_02

## Checklist

- ✔️ I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- ✔️ I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- ✔️ I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- ✔️ I checked that files in the `assets` folder are used by the extension itself
- ✔️ I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/coinpaprika/src/coinpaprika.tsx [hunk 1]
```diff
@@ -25,32 +51,121 @@ export default function Main() {
...
+          <ListFavoriteAction showFavorites={showFavorites} onToggle={onShowFavoritesToggle} />
+        </ActionPanel>
+      }
+    />
+  );
+}
+
+function FavoriteAction(props: FavoriteProps) {
+  const { isFavorite, onToggle } = props;
+  const title = isFavorite ? "Remove from Favorites" : "Add to Favorites";
```
Comment (pernielsentikaer) [inline extensions/coinpaprika/src/coinpaprika.tsx:161]:
```suggestion
  const title = isFavorite ? "Remove From Favorites" : "Add to Favorites";
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @gumbleliks 👋 

Thanks for your contribution 🔥 

I have some feedback for you 🙂

- Your metadata images are not formatted correct, we now have a tool build into Raycast that can help you make some [awesome good looking metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots).

- Maybe we should consider removing #1... in the title? Holding `⌘` down shows you some of the numbers - is there a reason for having them?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by gumbleliks** [comment]
> Hi @gumbleliks 👋
> 
> Thanks for your contribution 🔥
> 
> I have some feedback for you 🙂
> 
> * Your metadata images are not formatted correct, we now have a tool build into Raycast that can help you make some [awesome good looking metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots).
> * Maybe we should consider removing #1... in the title? Holding `⌘` down shows you some of the numbers - is there a reason for having them?
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Hi @pernielsentikaer,

thanks for your feedback and the hints.

-  I created a new metadata image with your awesome screenshot tool. I'll use it for further development 😊
Should I also replace the media Images with screenshots out of the raycast screenshot-tool or are they fine?

- Regarding the #<number> in the  title: The reason why I'm showing the number is the marketrank (measured in total market value of a cryptocurrency's circulating supply.) of the coins. So it is a quite needed information.

**Comment by gumbleliks** [comment]
> Hi @gumbleliks 👋
> 
> Thanks for your contribution 🔥
> 
> I have some feedback for you 🙂
> 
> * Your metadata images are not formatted correct, we now have a tool build into Raycast that can help you make some [awesome good looking metadata images](https://developers.raycast.com/basics/prepare-an-extension-for-store#screenshots).
> * Maybe we should consider removing #1... in the title? Holding `⌘` down shows you some of the numbers - is there a reason for having them?
> 
> I'm looking forward to testing this extension again 🔥
> 
> Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

Hi @pernielsentikaer,

thanks for your feedback and the hints.

I created a new metadata image with your awesome screenshot tool. I'll use it for further development 😊
Should I also replace the media Images with screenshots out of the raycast screenshot-tool or are they fine?

Regarding the # in the title: The reason why I'm showing the number is the marketrank (measured in total market value of a cryptocurrency's circulating supply.) of the coins. So it is a quite needed information.

**Pull request merged**